app/api/files/files.ts

```diff
@@ -9,7 +9,7 @@ import { FileType } from '../../shared/types/fileType';
 export const files = {
   async save(file: FileType) {
     const savedFile = await model.save(await validateFile(file));
-    await search.indexEntities({ sharedId: file.entity }, '+fullText');
+    await search.indexEntities({ sharedId: file.entity || savedFile.entity }, '+fullText');
```

Comment:
Additionally to the visualization problem, it was missing to index the entities of the updated files. I've added a test to ensure this indexation.

Reply:
@mfacar is `file.entity` ever used?

Reply:
Given the saved entity will always to have the entity value, I've removed file.entity